openedx/core/djangoapps/user_api/tests/test_views.py

```diff
@@ -804,17 +807,6 @@ def setUp(self):
         super(RegistrationViewValidationErrorTest, self).setUp()
         self.url = reverse("user_api_registration")
 
-    def _retireRequestUser(self):
-        """
-        Very basic user retirement initiation, logic copied form DeactivateLogoutView.  This only lands the user in
-        PENDING, simulating a retirement request only.
```

Comment:
This method was folded into `fake_requested_retirement()` in a more common location.